sass/main.scss

```diff
@@ -588,4 +595,53 @@ footer {
     .index-listing {
         max-width: none;
     }
-}
\ No newline at end of file
+}
+
+
+/* Anchor links */
+
+h1, h2, h3, h4, h5, h6  {
+  .zola-anchor {
+    all: initial;
+    visibility: hidden;
+  }
+
+  &:hover .zola-anchor {
+    all: initial;
+    cursor: pointer;
+    visibility: visible;
+  }
+
+  .zola-anchor-link {
+    vertical-align: middle;
+    display: inline-block;
+    fill: var(--text-color);
+  }
+}
+
+/* Inline code blocks */
+
+p > code {
+  padding: 0.2em 0.4em;
+  font-size: 85%;
+  background-color: var(--code-background);
+  border-radius: 6px;
+}
+
+/* Dark mode */
+
+@media (prefers-color-scheme: dark) {
+    :root {
+        --background-color: rgb(24, 24, 24);
+        --header-background-color: rgb(28, 28, 28);
+        --text-color: rgb(217, 217, 217);
+        --text-accent-color: rgb(130, 130, 130);
+        --border-color: rgb(66, 66, 66);
+        --link-color: rgb(71, 160, 247);
+        --code-background: rgb(32, 32, 32);
+    }
+
+    .img-dark {
+        filter: invert(100%) hue-rotate(180deg);
+    }
```

Comment:
What is this new `img-dark` class used for, btw?

Reply:
`img-dark` denotes an image can have the colours inverted in dark mode, I used it mostly for diagrams. I thought I added a parameter to the `img` shortcode but looks like I forgot 🥲